spec/features/idv/flow_spec.rb

```diff
@@ -348,18 +348,59 @@
 
     scenario 'pick USPS address verification' do
       sign_in_and_2fa_user
-
+      screenshot_and_open_image
       visit verify_session_path
 
       fill_out_idv_form_ok
       click_idv_continue
       fill_out_financial_form_ok
       click_idv_continue
       click_idv_address_choose_usps
-      click_idv_continue
+      click_on t('idv.buttons.send_letter')
 
       expect(current_path).to eq verify_review_path
     end
+
+    context 'cancel from USPS/Phone verification screen' do
+      context 'without js' do
+        it 'returns user to profile path' do
+          sign_in_and_2fa_user
+
+          loa3_sp_session
+
+          visit verify_session_path
+
+          fill_out_idv_form_ok
+          click_idv_continue
+          fill_out_financial_form_ok
+          click_idv_continue
+
+          click_idv_cancel
+
+          expect(current_path).to eq(profile_path)
+        end
+      end
+
+      context 'with js', js: true do
+        it 'redirects to profile from a modal' do
+          sign_in_and_2fa_user
+
```

Comment:
I _personally_ don't like having the first five lines of the code all jammed up there with no spaces, but I am totally willing to adhere to the conventions we already have.